MekHQ/src/mekhq/gui/dialog/ContractMarketDialog.java

```diff
@@ -242,11 +244,21 @@ private void initComponents() {
                 row.add("");
                 row.add(c.getType());
             }
+
+            final JumpPath path = campaign.calculateJumpPath(campaign.getCurrentSystem(), c.getSystem());
```

Comment:
I feel like we already calculate the jump path for a contract at some point. Given that it's a fairly involved operation, maybe we should just pre-record this data at the time the contract is created instead?

Reply:
That's probably a good idea, but definitely beyond the scope of this PR. I'll open an issue for it once this is merged, and probably handle it pretty quickly after.